dashboard/src/pages/packages/index.vue

```diff
@@ -124,9 +170,27 @@ const toggleLegend = () => {
                 'page-link': true,
                 disabled: !packageStore.hasNextPage,
               }"
+              aria-label="Go to next page"
               @click.prevent="packageStore.nextPage"
-              >Next</a
             >
+              Next <IconCaretRightFill />
+            </a>
+          </li>
+          <li v-if="packageStore.pager.total > packageStore.pager.maxPages">
+            <a
+              href="#"
+              :class="{
+                'page-link': true,
+                disabled:
+                  packageStore.pager.current == packageStore.pager.total,
+              }"
+              aria-label="Go to last page"
+              @click.prevent="
+                packageStore.fetchPackages(packageStore.pager.total)
+              "
+            >
+              Last <IconSkipEndFill />
```

Comment:
I think the functionality is clearer with the words, rather than icons only.  Having the text also means the button is larger and easier to click.  That said I'm okay with dropping the text if you feel strongly about it.